Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -439,6 +445,16 @@ void getSupportedTypes(HashSet<String, ASCIICaseInsensitiveHash>& types) const f
     m_visible = visible;
     if (m_visible)
         acceleratedRenderingStateChanged();
+
+#if PLATFORM(VISION)
+    if ([m_sampleBufferDisplayLayer respondsToSelector:@selector(setSTSLabel:)]) {
+        if (!visible) {
+            AVAudioSession *session = [PAL::getAVAudioSessionClass() sharedInstance];
+            [m_sampleBufferDisplayLayer setSTSLabel:session.spatialTrackingLabel];
```

Comment:
This is behind `PLATFORM(VISION)` because `session.spatialTrackingLabel` is only available there